editor/test/integration/test_util.clj

```diff
@@ -1277,6 +1277,9 @@
 (defmethod edit-resource-node "collisionobject" [resource-node-id]
   (g/update-property resource-node-id :linear-damping type-preserving-add 0.1))
 
+(defmethod edit-resource-node "compute_program" [resource-node-id]
+  (g/update-property resource-node-id :constants update-in [0 :name] str "new_name"))
```

Comment:
Let's use `\_` instead of "new_name" here, because this will append `new_name` to the existing string with each call. In other places, like for `"collection"` above, we append a `_`.